### PR TITLE
 ✨ move keycloak url to be an env var

### PR DESCRIPTION
**Pull request by wei-lee**
ping @TommyJ1994

### CODE REVIEW

File: projects/keycloak/exported-realm.json [hunk 1]
```diff
@@ -0,0 +1,3175 @@
+[ {
```
Comment (tomjackman) [inline projects/keycloak/exported-realm.json]: Was there any changes to the [secure-app realm](https://github.com/feedhenry/mobile-security/blob/master/projects/keycloak/secure-app-realm.json) or is this new realm file being used too? 

I see this config is also bringing in the master realm which we will cause the import to fail with a duplicate realm id.

**Review by tomjackman — COMMENTED** [review]
Small comment, but looks good apart from that.

File: projects/keycloak/exported-realm.json (hunk 1 above)
Comment (wei-lee) [reply projects/keycloak/exported-realm.json]: My mistake. This file wasn't meant to checked in. Will remove it.

**Review by tomjackman — APPROVED** [review]
LGTM :+1:

**Pull request merged**